Spec/RestClient.swift

```diff
@@ -839,6 +839,7 @@ class RestClient: QuickSpec {
                         }
                         
                         expect(testHTTPExecutor.requests).to(haveCount(Int(1 + options.httpMaxRetryCount)))
+                        expect((testHTTPExecutor.requests.count) < (_fallbackHosts.count + 1)).to(beTrue())
```

Comment:
Agree, I'm using local variable `_fallbackHosts` and its `count` property. 
Assert based on that hard-coded array is useless, because who knows, how big an array can be. 

If I use assert based on `options.fallbackHosts.count` like that: 
`expect((testHTTPExecutor.requests.count) < (options.fallbackHosts.count + 1)).to(beTrue())` 

I can check that count of executed requests (which is 5 = 1 to primary host + 4 to fallback hosts, cuz I use `httpMaxRetryCount = 4` value) is less than max available count (which is 6 = 1 to primary + 5 to fallback hosts).

So I just wanted to be sure that reduce in retry count from 5 to 4 - works as expected, does it make sense @mattheworiordan ?